package.json

```diff
@@ -105,14 +104,14 @@
     "bundlewatch": "bundlewatch",
     "start": "node scripts/start",
     "start:local": "node scripts/launch-local",
-    "start:mainnet": "cross-env ARAGON_ETH_NETWORK_TYPE=main npm start",
+    "start:mainnet": "cross-env ARAGON_ETH_NETWORK_TYPE=ethereum npm start",
```

Comment:
I've started renaming our internal references to "main" or "mainnet" to "Ethereum", as "mainnet" is not descriptive enough (e.g. xDai has a mainnet, Aragon Chain will have a mainnet).